wemix/bind/structs.go

```diff
@@ -208,6 +218,7 @@ func DeployGovContracts(opts *bind.TransactOpts, backend IBackend, optionDomains
 	}
 
 	// setup registry
+	logger.Info("Setting registry...")
 	txs = make([]*types.Transaction, 0)
 	if tx, err := gov.Registry.SetContractDomain(opts, metclient.ToBytes32("GovernanceContract"), gov.address.Gov); err != nil {
```

Comment:
You can use the code below.

	optionDomains["GovernanceContract"] = gov.address.Gov
	optionDomains["Staking"] = gov.address.Staking
	optionDomains["BallotStorage"] = gov.address.BallotStorage
	optionDomains["EnvStorage"] = gov.address.EnvStorage

	for name, address := range optionDomains {
		if tx, err := gov.Registry.SetContractDomain(opts, metclient.ToBytes32(name), address); err != nil {
			return nil, nil, errors.Wrap(err, fmt.Sprintf("SetContractDomain(%s)", name))
		} else {
			txs = append(txs, tx)
		}
	}